app/controllers/bibliographic_controller.rb

```diff
@@ -158,7 +160,7 @@ def voyager_opts
     # @param opts [Hash] optional arguments
     # @return [Array<Object>] the set of bib. records
     def find_by_id(id, opts)
-      VoyagerHelpers::Liberator.get_bib_record(id, nil, opts)
+      bib_adapter.get_bib_record(sanitized_id, nil, opts)
```

Comment:
If we're not going to use the id param I guess we should remove it from the argument list.